packages/headless/src/app/engine.test.ts

```diff
@@ -33,6 +34,7 @@ describe('engine', () => {
       options.configuration.organizationId
     );
     expect(configuration.platformUrl).toBe(options.configuration.platformUrl);
+    expect(configuration.name).toBe(options.configuration.name);
```

Comment:
I would have a separate test for this condition

Reply:
ok I will do that.